csv/ParameterDefs.csv

```diff
@@ -1,17 +1,5 @@
+DOC:Definition,,"System internal name of the parameter.  

Parameter are allowed to have the same name, as long as the attributes are different.  The HID and ID columns determine uniqueness.","Parameter Definition Identifier

This is the ""global"" identifier for the parameter.  It should be UNIQUE and STATIC among all parameters as it will be used to identify this parameter within other spreadsheets / contexts.  There is outstanding debate as to if this ID should be a simple incrementing ID (as below) or a computable SHA style identifier.",Member of the code set ParameterType,,"The encoding of the value.  Typically an intrinsic type.  'opaque' indicates an undefined value encoding and is stored as raw bits (blob)

If the Parameter Type is 'category', the value encoding should match the encoding of the KEY","If the Parameter Type is ""category"", this column holds the definition of the key:value map of meanings

""Code Set"" is the general term for an Enumeration

Remember to have the fill value as a category",From UDUNITS vocabulary,"A value used to represent missing or undefined data. Please ensure that the fill_value is NOT a valid value for the parameter. For example, 0 is typically a very poor fill_value since it is usually valid (i.e. for temp, lat, lon, pressure, salinity, etc).",The Expected display name for R2.1 and beyond. Will be a required field in R2.1 and beyond. May not be Used in 2.0,"This column should contain the decimal precision that the UI Charts/Tables will utilize to display the appropriate values.

If a precision is not known, leave it blank (it will still use the default).  If the default is to be used (currently precision of 5 decimal digits), and its approved to use whatever the default value is, set the value to ""default"".  If the value is not a float, the value should be set to 0.",Whether the parameter should be visible to the UI,The ID to the parameter function that the function executes,A key-value mapping using JSON or Python dictionary syntax. The key represents the function's argument and the value is the parameter name to be used as input. Values that are named CC_xxx represent calibration coefficients and are dynamically created at preload when they are found. Any parameter named CC_xxx will be dynamically created as a quanity type float32 with no metadata.,,The CF Standard Name for this parameter: http://cf-pcmdi.llnl.gov/documents/cf-standard-names/standard-name-table/20/cf-standard-name-table.html,OOI Data Product Identifier as defined in SAF. This column is required to apply QC and calibrations to data products.,A human-readable description of the parameter.,"CF Long Name. A long descriptive name, which may be used, for example, for labeling plots.",,
```

Comment:
The only edits I have introduced are in the list described in the commit text. Not sure how that happened.